x-pack/test/functional/services/ml/job_selection.ts

```diff
@@ -23,5 +23,18 @@ export function MachineLearningJobSelectionProvider({ getService }: FtrProviderC
         `Job selection should display jobs or groups '${jobOrGroupIds}' (got '${actualJobOrGroupLabels}')`
       );
     },
+
+    async assertJobSelectionNotContains(jobOrGroupId: string) {
+      const selectedJobsOrGroups = await testSubjects.findAll(
+        'mlJobSelectionBadges > ~mlJobSelectionBadge'
+      );
+      const actualJobOrGroupLabel = await Promise.all(
```

Comment:
Nit: there could still be multiple actual labels, even though we're only validating that a single one is not present.